packages/blaze/dombackend.js

```diff
@@ -9,13 +9,32 @@ if (! $jq)
 
 DOMBackend._$jq = $jq;
 
+
+DOMBackend.getContext = function() {
+  if (DOMBackend._context) {
+    return DOMBackend._context;
+  }
+  if ( DOMBackend._$jq.support.createHTMLDocument ) {
+    DOMBackend._context = document.implementation.createHTMLDocument( "" );
+
+    // Set the base href for the created document
+    // so any parsed elements with URLs
+    // are based on the document's URL (gh-2965)
+    const base = DOMBackend._context.createElement( "base" );
+    base.href = document.location.href;
+    DOMBackend._context.head.appendChild( base );
+  } else {
+    DOMBackend._context = document;
+  }
+  return DOMBackend._context;
```

Comment:
If `DOMBackend._context` exists it is already returned at line 14, therefore this line would return `undefined` right? In such case it should return `undefined` so it's immediately clear that nothing is returned.

Reply:
It gets set on lines 25 or 27. I think I just extracted this whole thing from jQuery